crates/mun_hir/src/ty/tests.rs

```diff
@@ -8,6 +8,18 @@ use mun_syntax::{ast, AstNode};
 use std::fmt::Write;
 use std::sync::Arc;
 
+#[test]
+fn comparison_not_implemented_for_struct() {
```

Comment:
Where is the corresponding `insta` file?

Reply:
Woops! Added